documents/operating.md

```diff
@@ -0,0 +1,216 @@
+Bad Idea Factory Limited Liability Corporation
+===================================================================
+
+Contents:
+
+1.  Official Operating Agreement
+2.  Partnership
+3.  Projects
+4.  Money
+5.  Governing Process
+6.  Policies
+
+---------------------------------------------------------------------
+SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
+---------------------------------------------------------------------
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?)
```

Comment:
I made a mistake in some places (fixed in an upcoming commit) -- the 70% (or whatever we decide) to pass a motion is actually 70% of *all* Corporate Overlords, not just those in attendance -- so not attending is a no vote (unless you vote via mail or some other format).